### DISCUSSION TITLE
How do I remove the holiday theme on Spriters Resource and sister websites?

### DISCUSSION THREAD

**Discussion by ChillyBeanBAM** [uBlockOrigin/uAssets#26534] [Q&A] [answered]
I am trying to find a way to remove the Christmas theme on https://www.spriters-resource.com (and its sister websites) shown in December and use its default one during that month for a good while, but I haven't been able to figure anything out. I've tried several different filters, but none of them are working.

**Comment by krystian3w** [comment, marked as answer]
Try network filtering:

```adb
/images/christmas/holiday/hat$image
/images/christmas/holiday/santa$image
/images/christmas/holiday/tree$image
/images/christmas/holiday/snow$image
```

It should works as long name of files and directories 📂📁 (folders/catalogues) are same.



---
---

Also possible block stars (mushrooms.png) and hat with CSS:

```adb
###header-logo::after
foo.bar##body:style(background-image: none !important; background-color: #cecece ! important;)
```

The `foo.bar` replace with known family domains separately by comma.

**Reply by ChillyBeanBAM** [reply]
> Try network filtering:
> 
> ```
> /images/christmas/holiday/hat$image
> /images/christmas/holiday/santa$image
> /images/christmas/holiday/tree$image
> /images/christmas/holiday/snow$image
> ```
> 
> It should works as long name of files and directories 📂📁 (folders/catalogues) are same.
> 
> Also possible block stars (mushrooms.png) and hat with CSS:
> 
> ```
> ###header-logo::after
> foo.bar##body:style(background-image: none !important; background-color: #cecece ! important;)
> ```
> 
> The `foo.bar` replace with known family domains separately by comma.

Wow, these worked! Thank you so much!

**Reply by uBlock-user** [reply]
or just `/images/christmas/$image`

**Reply by stephenhawk8054** [reply]
`/images/christmas/` will make it regex. Better to add `*` at the end to prevent it: `/images/christmas/*$image`
